src/OutSystems/GridAPI/Filter.ts

```diff
@@ -21,11 +21,12 @@ namespace OutSystems.GridAPI.Filter {
      * @returns {*}  {string} Return Message Success or message of error info if it's the case.
      */
     export function Search(gridID: string, searchedValue: string): string {
-        PerformanceAPI.SetMark('Filter.search');
```

Comment:
@glundgren93 Why have we changed this to Performance? Just to understand the rationale

Reply:
to leave it inside GridAPI

Reply:
Since we currently use this only in that scope, yes. But I don't have much context about that functionality. 
@rugoncalves @joselrio @JoaoFerreira-FrontEnd  any insight?

Reply:
If we do not encapsulate it in the GridAPI context we'll end up on blend (PerformanceAPI) with other different namespaces. This way we will enclose Performance inside the GridAPI scope.